library-core/src/main/java/com/mikepenz/fastadapter/adapters/IItemList.java

```diff
@@ -0,0 +1,46 @@
+package com.mikepenz.fastadapter.adapters;
+
+import android.support.v7.util.ListUpdateCallback;
+import com.mikepenz.fastadapter.IItem;
+
+import java.util.Collection;
+import java.util.List;
+
+public abstract class IItemList<Item extends IItem> {
+  protected ListUpdateCallback listCallback;
+
+  void setListUpdateCallback(ListUpdateCallback listCallback) {

```

Comment:
Providing the callback in the implementation constructor would give a bit more flexibility.

Reply:
Moving this into the constructor isn´t possible, but adding this method to the interface is.

Reply:
See note above for why this shouldn't be in the interface.